pkg/query-service/app/querier/v2/querier.go

```diff
@@ -478,7 +479,115 @@ func (q *querier) runClickHouseQueries(ctx context.Context, params *v3.QueryRang
 	return results, errQueriesByName, err
 }
 
+type logsListTsRange struct {
+	Start int64
+	End   int64
+}
+
+const HOUR_NANO = int64(3600000000000)
+
+func getLogsListTsRanges(start, end int64) []logsListTsRange {
```

Comment:
It seems like this doesn't need to exist here again. On a side note, I am finding you are forced to copy a lot of code to querier v2 even though it's not needed. Let's create a separate issue to address this.

Reply:
Do you think utils will be a right pace for this function ?

for the second part https://github.com/SigNoz/signoz/issues/new

Reply:
Yes, there is no specific reason why this should live here then it can go in utils.